main/management/commands/start.py

```diff
@@ -31,9 +32,22 @@ def handle(self, *args, **options):
             logging.error('Database could not be found, exiting.')
             sys.exit(1)
 
-        print('Running migrations')
-        call_command("migrate")
-        print('Running setup')
+        attempts_left = 5
+        while attempts_left:
+            try:
+                logging.info('Trying to run migrations')
+                call_command("migrate")
+                logging.info('Migrations ran')
```

Comment:
Okay, they ran. How did it go?

A better way to phrase this might be `Migrations successfully run!` or `Migrations complete.`